vmdb/app/helpers/application_helper.rb

```diff
@@ -1758,7 +1758,7 @@ def set_edit_timer_from_schedule(schedule)
       @edit[:new][:timer_weeks]  = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "weekly"
       @edit[:new][:timer_days]   = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "daily"
       @edit[:new][:timer_hours]  = schedule.run_at[:interval][:value] if schedule.run_at[:interval][:unit] == "hourly"
-      t                          = schedule.run_at[:start_time].to_time(:utc).in_time_zone(@edit[:tz])
+      t                          = schedule.run_at[:start_time].utc.in_time_zone(@edit[:tz])
```

Comment:
Can we do something about the fact that `schedule.run_at[:start_time]` is sometimes a String and sometimes a Time? It seems that would make this simpler.